woocommerce/woocommerce-ios
# [Store creation] Add progress view like web

Closes: #9362


## Description

Adds a loading screen for the store creation progress.

The store creation progress takes a long time and we have been displaying a basic loading screen. As a first iteration to improve the waiting experience, we decided to mimic the web experience by adding a progress bar and text. 
P2 - pecCkj-tX-p2#loading-screen

## Testing instructions

- Launch the app
- Tap on `Get Started`
- Tap `Log in` and login using your WPCOM account
- Enter any store name and tap `Continue`
- Tap `Try For Free` and you can see the new loading screen appear

## Screenshots

https://user-images.githubusercontent.com/524475/230277770-5d0cd4d6-3319-462d-803a-573087b34018.mp4

| Dark | Light |
|--------|--------|
| ![Simulator Screen Shot - iPhone 14 - 2023-04-06 at 10 52 32](https://user-images.githubusercontent.com/524475/230279168-3ea7cf63-904f-4c71-917c-2617ebddfe20.png) | ![Simulator Screen Shot - iPhone 14 - 2023-04-06 at 10 52 42](https://user-images.githubusercontent.com/524475/230279180-06fc12ac-f7f3-46cb-a16e-798a46d499e6.png) | 

---
